mako.5.scd

```diff
@@ -47,6 +47,23 @@ _none_, _dismiss_, _dismiss-all_, _dismiss-group_, _invoke-default-action_.
 *on-touch*=_action_
 	Default: dismiss
 
+*on-notify* = _action_
+	Default: none
+
+When _exec_ is used, the command will be executed in a POSIX shell. The
+shell variable _id_ will be set to the notification ID. For example, the
+following option will display an interactive action menu on middle click:
+
+```
+on-button-middle=exec makoctl menu -n "$id" dmenu -p 'Select action: '
+```
+
+For example, the following option will play a sound when a new notification
```

Comment:
This second "for example" reads a bit weird since there was one above, can probably just remove it.

Reply:
Done